src/shared/watchdog.c

```diff
@@ -126,6 +139,8 @@ int watchdog_ping(void) {
         r = ioctl(watchdog_fd, WDIOC_KEEPALIVE, 0);
         if (r < 0)
                 return log_warning_errno(errno, "Failed to ping hardware watchdog: %m");
+        else
```

Comment:
in the interest of keeping indentation levels minimal, just drop the "else". Since the first if branch returns in all cases this will not change behaviour